substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -129,13 +129,25 @@ pub struct StorageParams {
 	#[arg(long, default_value = "false")]
 	pub disable_pov_recorder: bool,
 
-	/// The batch size for the write benchmark.
+	/// The batch size for the read/write benchmark.
 	///
 	/// Since the write size needs to also include the cost of computing the storage root, which is
 	/// done once at the end of the block, the batch size is used to simulate multiple writes in a
 	/// block.
 	#[arg(long, default_value_t = 100_000)]
 	pub batch_size: usize,
+
+	/// Run the benchmark on block validation.
+	///
+	/// PoV recorder must be activated to provide a storage proof for block validation at runtime.
+	#[arg(long, default_value_t = false, conflicts_with = "disable_pov_recorder")]
+	pub on_block_validation: bool,
+
+	/// Number of rounds to run the benchmark on block validation.
+	///
+	/// We need to run the benchmark several times to avoid fluctuations during runtime setup.
+	#[arg(long, default_value_t = 5, requires = "on_block_validation")]
+	pub on_block_validation_rounds: u32,
```

Comment:
This should also be relevant for testing the dbs? Like there we should also do this multiple times. And also 5 should be increased to at least 20.

Reply:
I don't think so. During normal runs, we measure every access, so it's already more or less okay. We should use it only with the runtime executions. That's why `on_block_validation` is required.